packages/odyssey-react-mui/src/labs/SideNav/SideNav.tsx

```diff
@@ -175,21 +155,28 @@ const SideNavListContainer = styled("ul")(() => ({
   listStyleType: "none",
 }));
 
-const SideNavScrollableContainer = styled("div")(() => ({
+const SideNavScrollableContainer = styled("div", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})(({ odysseyDesignTokens }: { odysseyDesignTokens: DesignTokens }) => ({
   flex: 1,
   overflowY: "auto",
+  paddingInline: odysseyDesignTokens.Spacing2,
+}));
+
+const SectionHeaderContainer = styled("li", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})(({ odysseyDesignTokens }: { odysseyDesignTokens: DesignTokens }) => ({
+  paddingBlock: odysseyDesignTokens.Spacing3,
+  paddingInline: odysseyDesignTokens.Spacing4,
 }));
 
-const SectionHeader = styled("li", {
+const SectionHeader = styled("h3", {
```

Comment:
Do we have an `li` somewhere else now?

Reply:
Yes, the `SectionHeaderContainer` is an `li`. This is targeting specifically the inner content of that.

This was part of a set of adjustments I made to ensure that the sidenav has the proper ARIA roles and a11y support; I had to adjust some of the markup because we had some children that had roles incompatible with their parents, specifically around `list` and `listitem`.